### PR TITLE
Edits to get faucet working with paloma

### PR DESCRIPTION
**Pull request by measure-fi**
https://github.com/palomachain/paloma/issues/205

## Background

This is not a full conversion, just the minimum amount needed to get faucet up-and-running.

## Testing Completed

- [ ] It works for the livia testnet

### CODE REVIEW

File: faucet.go [hunk 1]
```diff
@@ -591,7 +547,7 @@ func main() {
 	mux.HandleFunc("/claim", createGetCoinsHandler(db))
 
 	c := cors.New(cors.Options{
-		AllowedOrigins:   []string{"https://faucet.terra.money", "http://localhost", "localhost", "http://localhost:3000", "http://localhost:8080"},
+		AllowedOrigins:   []string{"https://faucet.paloma.money", "http://localhost", "localhost", "http://localhost:3000", "http://localhost:8080"},
```
Comment (Vizualni) [inline faucet.go:550]: we have domain `paloma.money`?

File: README.md [hunk 2]
```diff
@@ -24,8 +24,10 @@ Run it with the mnemonic and recaptcha key as env vars.
 
 ```bash
 docker run -p 3000:3000 \
-    -e MNEMONIC=$MY_MNEMONIC \
-    -e RECAPTCHA_KEY=$RECAPTCHA_KEY \
-    -e PORT=8080 \  # default to 3000
+    -e MNEMONIC="$(cat mnemonic.txt)" \
```
Comment (Vizualni) [inline README.md:27]: Shall we also rename other occurrences of the word terra in this project?

File: faucet.go (hunk 1 above)
Comment (measure-fi) [reply faucet.go:550]: Seems unlikely.

Comment (verabehr) [reply faucet.go:550]: we don't. Terra had a lot under their terra.money domai, not just the faucet. I think faucet.palomachain.com would work for us. - cc @taariq

Comment (taariq) [reply faucet.go:558]: Let's encrypt?

File: faucet.go [hunk 3]
```diff
@@ -59,39 +56,29 @@ var cdc *params.EncodingConfig
...
+	accountPubKeyPrefix    = "palomapub"
+	validatorAddressPrefix = "palomavaloper"
+	validatorPubKeyPrefix  = "palomavaloperpub"
+	consNodeAddressPrefix  = "palomavalcons"
+	consNodePubKeyPrefix   = "palomavalconspub"
 )
 
 var amountTable = map[string]int64{
-	app.BondDenom: 5 * MicroUnit,
+	"ugrain": 5 * MicroUnit,
```
Comment (measure-fi) [inline faucet.go:71]: @taariq You want this to be `100000grain`, correct?

**Review by Vizualni — APPROVED** [review]
lgtm

**Pull request merged**